index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Odongo full-stack web-developer</title>
+</head>
+<body>
+    <link rel="stylesheet" href="style.css">
+    <header class="header">
+        <nav class="header-nav">
+            <p  class="MyLogo">
+                <a class="nav-brand" href="">My Logo</a>
+            </p>
+            
+            <ul class="topNav">
+                <li class="nav-items">
+                    <a class="nav-link" href="#Portfolio">Portfolio</a>
+                </li>
+                <li class="nav-items">
+                    <a class="nav-link" href="#About">About</a>
+                </li>
+                <li class="nav-items">
+                    <a class="nav-link" href="#Contact">Contact</a>
+                </li>
+            </ul>
+
+            <div class="hamburger">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
+        </nav>
+    </header>
+    <main>
+        <section class="box2">
+            <div class="Greetings">
+                <h1>I'm Henry<br>Glad to see you!</h1>
+            </div>
+         <p class="Content">
+            I'm a software developer! I can help you build a product,feature or website Look through some of my work and  experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+         </p>
+
+         <p class="contact">LET'S CONNECT</p>
```

Comment:
As requested by the previous reviewer, please make sure to use a `h3` tag instead of a `p`. You could have:
`<h3 class="contact">LET'S CONNECT</h3>`